xml_converter/src/string_helper.cpp

```diff
@@ -142,49 +162,83 @@ std::string base64_encode(uint8_t const* buf, unsigned int bufLen) {
   return ret;
 }
 
+#include "function_timers.hpp"
+
+// TODO: Write Tests For This
+static unsigned char base64_lookup[256] = {
```

Comment:
This table mirrors the one used in the normalize_lookup. Does it follow the same mapping as in that table?

Reply:
This absolutely does **NOT** mirror the `normalize_lookup` table. This table maps an ASCII character to an integer value or an error value (255), `normalize_lookup` maps an ASCII character to another ASCII character or null/ignore value. This table supports all base64 characters which include two symbols `+` and `/`. Those characters are ignored in the normalize table.

Reply:
Can you please add a comment in the code similar to line 59 that describes the mapping for this table?

Reply:
Done